src/test/ui/traits/trait-suggest-where-clause.stderr

```diff
@@ -12,7 +12,7 @@ LL |     mem::size_of::<U>();
 LL | pub const fn size_of<T>() -> usize {
    |                      - required by this bound in `std::mem::size_of`
    |
-   = help: the trait `std::marker::Sized` is not implemented for `U`
+   = help: type parameters have an implicit `Sized` requirement by default
```

Comment:
this is a case where it kind of makes sense, but I feel like we would ideally want it to be in place of the "required by this bound" message. e.g., something like

```
pub const fn size_of<T>
                     - required because of the default `Sized` requirement on this type parameter
```